source/Junkership.js

```diff
@@ -152,7 +153,8 @@ export default class Junkership extends Pixi.Sprite {
     }
 
     changePowerUp(newPowerUp) {
-        this.powerUp = newPowerUp
+        this.powerUp = this.WeaponList[newPowerUp]
+        console.log(this.powerUp)
```

Comment:
I'm still eye-ing up our use of `console.log` everywhere. I understand we use it to prove our functionality is happening, but we should eventually be sure to remove them.

<img width="603" alt="screen shot 2016-02-26 at 12 34 09 pm" src="https://cloud.githubusercontent.com/assets/1202938/13364714/51351700-dc85-11e5-94c9-83dfebd835d5.png">

This sort of logging will only snowball from here if we don't manage it.